nixos/tests/ctrld.nix

```diff
@@ -0,0 +1,25 @@
+
+import ./make-test-python.nix ({ pkgs, ... }: {
+  name = "ctrld";
+
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ arthsmn ];
+  };
+
+  nodes = {
+    machine = { pkgs, ... }: {
+      environment.systemPackages = [ pkgs.dig ];
+    };
```

Comment:
unused?

Either test something like `machine.wait_until_succeeds("dig verify.controld.com @client +short")` or

```suggestion
```

Reply:
Like this?